### PR TITLE
feat: validate name as a required value

### LINKED ISSUE #1302: Mark Name property in add link as required

### PR DESCRIPTION
**Pull request by LinaYahya**
closes #1302

### CODE REVIEW

File: cypress/e2e/item/create/createLink.cy.ts [hunk 1]
```diff
@@ -86,5 +87,25 @@ describe('Create Link', () => {
...
+    it('cannot have an empty name', () => {
+      const FOLDER = PackedFolderItemFactory();
+      cy.setUpApi({ items: [FOLDER] });
+      const { id } = FOLDER;
+
+      // go to children item
+      cy.visit(buildItemPath(id));
+
+      // create
+      createLink(LINK_ITEM_WITH_BLANC_NAME, {
```
Comment (spaenleh) [inline cypress/e2e/item/create/createLink.cy.ts:100]:
```suggestion
      createLink(LINK_ITEM_WITH_BLANK_NAME, {
```

File: yarn.lock
Comment (spaenleh) [inline yarn.lock:1]: I am not sure why you have changes in yarn.lock ?

**Review by spaenleh — APPROVED** [review]
Looks good, thank you for this fix. Just a typo and a question about updating yarn.lock.

File: yarn.lock
Comment (LinaYahya) [reply yarn.lock:1]: Even me, But there was sth broken till I updated yarn

**Comment by spaenleh** [comment]
@LinaYahya I was able to revert the changes to the lock file. Looks like nothing broke (at least the CI passes). I also reverted the merge commit and rebased on main (I prefer it like that as it makes subsequent rebases easier). In github use the dropdown next to "Update branch" and select "Update with rebase instead" 😄

**Pull request merged**